### PR TITLE
chore: migrate error messages on descriptions

### PR DESCRIPTION
**Pull request by anshumanv**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [x] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?
NA

### Motivation / Use-Case
Error messages are unused.

### Breaking Changes
No

### CODE REVIEW

File: lib/options.json [hunk 1]
```diff
@@ -205,13 +206,16 @@
...
     },
     "compress": {
-      "type": "boolean"
+      "type": "boolean",
+      "description": "https://webpack.js.org/configuration/dev-server/#devservercompress"
     },
     "dev": {
-      "type": "object"
+      "type": "object",
+      "description": "https://webpack.js.org/configuration/dev-server/#devserverdev-"
```
Comment (snitin315) [inline lib/options.json:218]: I think we should use proper descriptions instead of links. Links are subject to change as we improve docs.

Comment (anshumanv) [reply lib/options.json:218]: yeah we could but these show up in validation errors which can give more visibility of the error on the detailed docs page, I do have proper desc in older commit we can revert to that if we want /cc @alexander-akait

Comment (alexander-akait) [reply lib/options.json:218]: We need keep description and link(s), only links is bad, especial for local usage, I don't want to open link for simple description

Comment (anshumanv) [reply lib/options.json:218]: yep lets do it

Comment (anshumanv) [reply lib/options.json]: done

**Comment by alexander-akait** [comment]
I will finish, also we need to add `description` for all properties, includes nested, for example `client.overlay`

**Comment by alexander-akait** [comment]
Thanks

**Pull request merged**